From 49186450ff33e4a75d966cee51bcb6446fae3639 Mon Sep 17 00:00:00 2001
From: Yuan Liao <liaoyuan@gmail.com>
Date: Sat, 3 Jun 2023 09:59:11 -0700
Subject: [PATCH 1/2] exm-backtrace.c: Explicitly include stdint.h for
 uintptr_t usage

The file uses type uintptr_t from stdint.h, which is indirectly included
by backtrace.h.  Even so, the file should unconditionally include
stdint.h just in case backtrace.h no longer includes stdint.h in the
future.

Signed-off-by: Yuan Liao <liaoyuan@gmail.com>
---
 src/exm-backtrace.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/exm-backtrace.c b/src/exm-backtrace.c
index 56b253a..4951394 100644
--- a/src/exm-backtrace.c
+++ b/src/exm-backtrace.c
@@ -21,6 +21,7 @@
 #include "exm-backtrace.h"
 
 #include <glib.h>
+#include <stdint.h>
 
 #include <backtrace-supported.h>
 #include <backtrace.h>
-- 
2.39.3

